zgw/zaken-api/src/main/kotlin/nl/nlportal/zakenapi/client/request/ZaakRollen.kt

```diff
@@ -13,8 +13,6 @@ interface SearchZaakRollen : PagedRetrieve<SearchZaakRollen, ZaakRol>, Authentic
     fun forZaak(zaakUrl: String): SearchZaakRollen
 
     fun forZaak(zaakId: UUID): SearchZaakRollen
-
-    fun ofVestigingsNummer(vestigingsNummer: String): SearchZaakRollen
```

Comment:
Please do not change the class signature in a hotfix. If we want to remove this then it needs to go as a separate PR into `next-major`.